### PR TITLE
Runbook: clarify MimirIngesterReachingSeriesLimit errors and retries

### PR DESCRIPTION
**Pull request by bboreham**
#### What this PR does

Suggested runbook text.

### CODE REVIEW

File: docs/sources/mimir/manage/mimir-runbooks/_index.md [hunk 1]
```diff
@@ -41,7 +41,15 @@ If nothing obvious from the above, check for increased load:
 
 ### MimirIngesterReachingSeriesLimit
 
-This alert fires when the `max_series` per ingester instance limit is enabled and the actual number of in-memory series in an ingester is reaching the limit. Once the limit is reached, writes to the ingester will fail (5xx) for new series, while appending samples to existing ones will continue to succeed.
+This alert fires when the `max_series` per ingester instance limit is enabled and the actual number of in-memory series in an ingester is reaching the limit.
```
Comment (tacole02) [inline docs/sources/mimir/manage/mimir-runbooks/_index.md:44]:
```suggestion
This alert fires when the `max_series` per ingester instance limit is enabled and the actual number of in-memory series in an ingester reaches that limit.
```

File: docs/sources/mimir/manage/mimir-runbooks/_index.md [hunk 2]
```diff
@@ -41,7 +41,15 @@ If nothing obvious from the above, check for increased load:
...
+The threshold is set at 80%, to give some chance to react before the limit is reached.
```
Comment (tacole02) [inline docs/sources/mimir/manage/mimir-runbooks/_index.md:45]:
```suggestion
The threshold is set at 80% to give the chance to react before the limit is reached.
```

File: docs/sources/mimir/manage/mimir-runbooks/_index.md [hunk 3]
```diff
@@ -41,7 +41,15 @@ If nothing obvious from the above, check for increased load:
...
+Once the limit is reached, writes to the ingester will fail for new series. Appending samples to existing ones will continue to succeed.
```
Comment (tacole02) [inline docs/sources/mimir/manage/mimir-runbooks/_index.md:46]:
```suggestion
After the limit is reached, writes to the ingester fail for new series. Appending samples to existing ones continue to succeed.
```

Comment (tacole02) [reply docs/sources/mimir/manage/mimir-runbooks/_index.md:46]: Can we avoid using "writes" as a noun? Could we say "write requests" or something else, if it's more accurate. We could then also say "Appending samples to existing requests continue to succeed" for greater clarity.

Comment (tacole02) [reply docs/sources/mimir/manage/mimir-runbooks/_index.md:46]: We avoid using "will" in the docs.

File: docs/sources/mimir/manage/mimir-runbooks/_index.md [hunk 4]
```diff
@@ -41,7 +41,15 @@ If nothing obvious from the above, check for increased load:
...
+
+Note that the error responses sent back to the sender are classed as "server error" (5xx), which should result in a retry by the sender.
```
Comment (tacole02) [inline docs/sources/mimir/manage/mimir-runbooks/_index.md:48]: classed = classified?
Also, could we say "server errors" for agreement?

File: docs/sources/mimir/manage/mimir-runbooks/_index.md [hunk 5]
```diff
@@ -41,7 +41,15 @@ If nothing obvious from the above, check for increased load:
...
+While this situation continues, these retries will stall the flow of data, and newer data will queue up on the sender.
```
Comment (tacole02) [inline docs/sources/mimir/manage/mimir-runbooks/_index.md:49]:
```suggestion
While this situation continues, these retries stall the flow of data, and newer data queues up on the sender.
```

File: docs/sources/mimir/manage/mimir-runbooks/_index.md [hunk 6]
```diff
@@ -41,7 +41,15 @@ If nothing obvious from the above, check for increased load:
...
+If the condition is cleared in a short time, service can be restored with no data loss.
```
Comment (tacole02) [inline docs/sources/mimir/manage/mimir-runbooks/_index.md:50]:
```suggestion
If the condition is cleared in a short time, service is restored with no data loss.
```

File: docs/sources/mimir/manage/mimir-runbooks/_index.md [hunk 7]
```diff
@@ -41,7 +41,15 @@ If nothing obvious from the above, check for increased load:
...
+
+This is different to what happens when the `max_global_series_per_user` is exceeded, which is considered a "client error" (4xx) where excess data is discarded.
```
Comment (tacole02) [inline docs/sources/mimir/manage/mimir-runbooks/_index.md:52]:
```suggestion
This is different to what happens when the `max_global_series_per_user` limit is exceeded, which is considered a "client error" (4xx). In this case, excess data is discarded.
```

**Review by tacole02 — APPROVED** [review]
Looks good! A few minor suggestions.

File: docs/sources/mimir/manage/mimir-runbooks/_index.md (hunk 1 above)
Comment (tacole02) [reply docs/sources/mimir/manage/mimir-runbooks/_index.md:44]: Oh, I see. Then, could we say `is close to reaching the limit` or 'nearly reaches the limit` ?

File: docs/sources/mimir/manage/mimir-runbooks/_index.md (hunk 6 above)
Comment (bboreham) [reply docs/sources/mimir/manage/mimir-runbooks/_index.md:50]: I left this one; it seems more correct to me the way I wrote it.
Maybe if you explain more what you dislike about it I can find another wording.

Comment (bboreham) [reply docs/sources/mimir/manage/mimir-runbooks/_index.md:50]: That is what I meant. Restoring service depends on nothing else going wrong.

**Pull request merged**